sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -111,23 +111,26 @@ catch (Exception ex)
 }
 ```
 
```

Comment:
This file is easier to look at just in [readme form](https://github.com/Azure/azure-sdk-for-net/tree/feature/adt/drwill/crudSnippet/sdk/digitaltwins/Azure.DigitalTwins.Core/samples).